galite-tests/src/test/kotlin/org/kopi/galite/tests/form/FormSample.kt

```diff
@@ -90,6 +92,10 @@ object FormSample : Form() {
     options(BlockOption.NOINSERT)
   }
 
+  val tb4ToTestChangeBlockAccess = insertBlock(TestBlock(), p1) {
+    changeBlockAccess(Access.ACS_SKIPPED, Modes.MOD_QUERY, Modes.MOD_INSERT)
```

Comment:
I think we should  find a better name for this functionality rather than changeBlockAccess